doc/ws.md

```diff
@@ -0,0 +1,121 @@
+# Websockets interface
+
+## What is this?
+
+This interface is a new way to talk with Orion using websockets, this is point to improve the comunication with a faster and efficient protocol, now we have a persistent connection with a open socket. 
+
+This feature is in a very early phase, at this moment you can create, delete, modify and update entities and you can subscribe for notifications, but is not well tested and up to date is only a wrapped version of REST API, there is not a binary protocol.
```

Comment:
but **it** is not well tested ... wrapped version of **the** REST API  ...  there is **no ** binary protocol **implemented yet**